manage-data/lifecycle/index-lifecycle-management/skip-rollover.md

```diff
@@ -9,17 +9,40 @@ products:
 
 # Skip rollover [skipping-rollover]
 
-When `index.lifecycle.indexing_complete` is set to `true`, {{ilm-init}} won’t perform the rollover action on an index, even if it otherwise meets the rollover criteria. It’s set automatically by {{ilm-init}} when the rollover action completes successfully.
+You may want to use {{ilm}} to manage your indices but do not want your indices to roll over automatically. For instance, if you're not working with continuously ingested data, you may prefer to create a new index on a regular interval, such as the first Monday of each month. In this case you may not want your indices to roll over, but you do still want to use an {{ilm-init}} policy to have the indices transition automatically through [lifecycle phases](/manage-data/lifecycle/index-lifecycle-management/index-lifecycle.md#ilm-phase-transitions) as they age.
 
-You can set it manually to skip rollover if you need to make an exception to your normal lifecycle policy and update the alias to force a roll over, but want {{ilm-init}} to continue to manage the index. If you use the rollover API, it is not necessary to configure this setting manually.
+You can configure indices to skip rollover either as part of an {{ilm-init}} policy or manually by adjusting the index settings.
+
+
+## Skip rollover through an {{ilm-init}} policy
+
+When you [configure a lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md) in {{kib}}, you can choose whether or not rollover is enabled.
+
+To update the automatic rollover setting in an {{ilm-init}} policy:
+
+1. Go to **Stack Management > Index Lifecycle Policies**.
+
+1. Select a policy to update, and from the **Actions** menu select **Edit**.
+
+1. In the **Hot phase** section, expand the advanced settings.
+
+1. Update the **Enable rollover** option to enable or disable automatic rollover.
+
+    ![Create policy page](/manage-data/images/elasticsearch-reference-create-policy.png "")
+
+
+## Skip rollover manually
+
+When the [`index.lifecycle.indexing_complete`](elasticsearch://reference/elasticsearch/configuration-reference/index-lifecycle-management-settings.md#_index_level_settings_2) setting is `true`, {{ilm-init}} won’t perform the rollover action on an index, even if it otherwise meets the rollover criteria. This setting is updated to `true` automatically by {{ilm-init}} when the rollover action completes successfully.
+
+You can update this setting manually to skip rollover if, for instance, you need to make an exception to your normal lifecycle policy and update the alias to force a rollover, but you want {{ilm-init}} to continue to manage the index. If you use the rollover API rather than an {{ilm-init}} policy to rollover indices, it is not necessary to configure this setting manually.
```

Comment:
```suggestion
You can update this setting manually to skip rollover if, for instance, you need to make an exception to your normal lifecycle policy and update the alias to force a rollover, but you want {{ilm-init}} to continue to manage the index. If you use the rollover API rather than an {{ilm-init}} policy to roll over indices, it is not necessary to configure this setting manually.
```
Is this because you can't automate rollover via the API?

Reply:
I'm not sure. I just moved this content from its original location. I gather that this is someone running the rollover API on a one-time basis, in which case it automatically updates the setting to "true".